oak/server/rust/oak_loader/src/main.rs

```diff
@@ -199,8 +200,15 @@ fn main() -> anyhow::Result<()> {
         read_to_string(&opt.grpc_tls_private_key).context("could not read gRPC TLS private key")?;
     let grpc_tls_certificate =
         read_to_string(&opt.grpc_tls_certificate).context("could not read gRPC TLS certificate")?;
-    let root_tls_certificate =
-        read_to_string(&opt.root_tls_certificate).context("could not read root TLS certificate")?;
+    let root_tls_certificate = match &opt.root_tls_certificate {
+        Some(certificate_path) if cfg!(feature = "oak_debug") => {
+            read_to_string(certificate_path).context("could not read root TLS certificate")?
+        }
+        Some(_) => bail!("Specifying `root-tls-certificate` requires the `oak_debug` feature."),
```

Comment:
Personally I think this would read better as an if inside a single branch, rather than as a match guard, but up to you. This catches my eye because it looks like it is ignoring something.

Reply:
Restructured it.